recipes-support/salt/salt_3000.2.bb

```
@@ -18,7 +18,7 @@ PACKAGECONFIG[tcp] = ",,python3-pycryptodome"
 PACKAGECONFIG[zeromq] = ",,python3-pycryptodome python3-pyzmq"
 
 SRC_URI = "\
-    git://github.com/ni/salt.git;protocol=https;branch=ni/master/3000.2 \
+    git://github.com/ni/salt.git;protocol=https;branch=ni/skyline-25.5/3000.2 \
```

```suggestion
    git://github.com/ni/salt.git;protocol=https;branch=ni/skyline-25.8/3000.2 \
```

I've created a branch for the 25.8 release

---

Thanks. Updated branch